### DISCUSSION TITLE
Is it possible to catch `foo(arg )`

### DISCUSSION THREAD

**Discussion by jeremy-rifkin** [eslint/eslint#16546] [Q&A] [answered]
Is there a rule for function call parameters along the lines of:
```ts
foo(arg); // ok
foo(arg ); // error
foo( arg); // error
foo( arg ); // error
foo(
    arg
); // ok
```

**Comment by mdjermanovic** [comment, marked as answer]
You can use [space-in-parens](https://eslint.org/docs/latest/rules/space-in-parens) rule, here's a [Playground demo](https://eslint.org/play/#eyJ0ZXh0IjoiZm9vKGFyZyk7IC8vIG9rXG5mb28oYXJnICk7IC8vIGVycm9yXG5mb28oIGFyZyk7IC8vIGVycm9yXG5mb28oIGFyZyApOyAvLyBlcnJvclxuZm9vKFxuICAgIGFyZ1xuKTsgLy8gb2siLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOiJsYXRlc3QiLCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7InNwYWNlLWluLXBhcmVucyI6WyJlcnJvciJdfSwiZW52Ijp7ImVzNiI6dHJ1ZX19fQ==).

**Comment by jeremy-rifkin** [comment]
Thanks!
